### ISSUE TITLE
Perf Dashboard: Test-picker broken in latest version (no test suite menu listed)

### ISSUE THREAD

**Issue by qyearsley** [catapult-project/catapult#1545]
Chrome version: 47.0.2516.0 (Linux)
URL: https://clean-qyearsley-3ce37cfa-dot-chromeperf.appspot.com/report

Please describe the problem:
When typing in a test suite name in the test-picker in the [latest version](https://clean-qyearsley-3ce37cfa-dot-chromeperf.appspot.com/report) (deployed 2015-09-29 afternoon), no candidate test suites are shown.

Additionally, when I type something then delete it, I get the following error in the console:
Uncaught TypeError: Cannot read property '' of undefined ([test-picker.html:141](https://github.com/catapult-project/catapult/blob/master/dashboard/dashboard/elements/test-picker.html#L141)).

This have been broken in the last week, because the version deployed last week (https://clean-qyearsley-46df66c4-dot-chromeperf.appspot.com/report) works.

Recent commits that affected test-picker:
https://github.com/catapult-project/catapult/commits/master/dashboard/dashboard/elements/test-picker.html

Showing the list of test-suites should not depend on any XHR, since (I believe) it's just based on the TEST_SUITES object. It appears that TEST_SUITES is not set as a property on the test-picker element, but is set as a global on the window object.

@chrisphn

**Comment by chrisphn** [comment]
Taking a look.

**Comment by anniesullie** [comment]
Was this found running manual tests during weekly deployment? If so, hurray for precess!

**Comment by qyearsley** [comment]
Yep! This time, if I pushed without checking, then someone else would have likely stumbled across this bug first, which would be bad and troublesome.

(Still: I wasn't as systematic/thorough as I could have been -- my process was just opening up all pages and trying to do a few things on each page. So it would have been possible for me to miss some types of bugs. This is good to keep in mind for next time I push.)

BTW, this is fixed now and the latest deployed version is after the fix. When checking after deploying the last time, I observed the warning "Failed to fetch graph data" (changed in https://codereview.chromium.org/1368703004/) on a couple graphs, but the graphs displayed correctly after reloading. I *believe* that this is intended behavior, and that previously I would have just seen a spinner.
